packages/squiggle-lang/benchmark/lib.ts

```diff
@@ -0,0 +1,42 @@
+import { run, squiggleExpression, errorValueToString } from "../src/js/index";
+import * as chalk from "chalk";
+
+let testRun = (x: string): squiggleExpression => {
+  let result = run(x, { sampleCount: 100, xyPointLength: 100 });
+  if (result.tag === "Ok") {
+    return result.value;
+  } else {
+    throw Error(
+      "Expected squiggle expression to evaluate but got error: " +
+        errorValueToString(result.value)
+    );
+  }
+};
+
+export function test(name: string, fn: () => void) {
+  console.log(chalk.cyan.bold(name));
+  fn();
+}
+
+export function expectEqual(expression1: string, expression2: string) {
```

Comment:
This function is a bit messy; but I'm okay going to a code quality 6 for this part of the codebase.